### ISSUE TITLE
[Feature Request] Display number of material left in network when hovering over materials in the crafting grid

### ISSUE THREAD

**Issue by tanyan-tanyan** [refinedmods/refinedstorage#2737]
Hi! I saw that the mod has a feature which shows you who modified an item in the grid, and how long ago. Having played around with AE2 in the past, I absolutely love that this is a thing as it's an upgrade from the former mod. I have a feature request for the instances when the user is crafting items using the crafting grid.

When crafting items such as the storage disks and other complicated items from some mods, I often don't have enough components at first and then I need to run back and forth between furnaces, smelters, infusers etc. I was wondering if it'd be possible to have an extra line display a number when hovering over an item in the crafting grid, which would tell the user how many items there are left in the network. This would be useful as it'd save many of us time spent on organizing all of the resources and figuring out what we're short on. The below picture depicts the tooltip I am talking about.

![CIefvsg](https://user-images.githubusercontent.com/68793071/98464306-da4bd000-21b9-11eb-9170-4ec5ebe47816.png)

So in summary, I'd like to request that this tooltip tells me how much silicon there is in my network, other than the silicon I'm using to make the storage part. Cheers!

**Comment by raoulvdberge** [comment]
This is a good idea. Do you have any suggestions on how to display the count in a clean way?

**Comment by Garueri** [comment]
Maybe you could put the items after the item name like
''Silicon (1259) or [2500]''
And maybe you could add a sort of calculator to divide the numbers in stacks? Like, this recipe uses 4 silicon, so the number would appear (64)[16]?
Im just dropping by, please don't ''listen to me'', I have no idea how to create mods

**Comment by Darkere** [comment]
Good suggestion tbh that would be my first thought as well. 

Another option would be to display the number in on the item. Like in small text, top left of the slot.

**Comment by raoulvdberge** [comment]
Hi! Thanks for the suggestion. This has been added to the RS 2 tracker, and will be implemented for RS 2.x. 

Closing this issue for RS 1.x on this repository.
